### PR TITLE
fix tidb-lightning errors

### PR DESCRIPTION
**Pull request by mightyguava**
fixes for https://github.com/pingcap/tidb-operator/commit/9a3b1e27603dd28deac8dae43e34d21e1a678629

### CODE REVIEW

File: charts/tidb-lightning/templates/rclone-conf.yaml [hunk 1]
```diff
@@ -1,10 +1,9 @@
+{{- if .Values.dataSource.remote.rcloneConfig }}
 apiVersion: v1
 kind: ConfigMap
 metadata:
   name: rclone-{{ include "tidb-lightning.name" . }}
-type: Opaque
```
Comment (mightyguava) [inline charts/tidb-lightning/templates/rclone-conf.yaml]: `type` is only allowed in `Secret`s

File: charts/tidb-lightning/templates/rclone-conf.yaml [hunk 2]
```diff
@@ -1,10 +1,9 @@
+{{- if .Values.dataSource.remote.rcloneConfig }}
```
Comment (mightyguava) [inline charts/tidb-lightning/templates/rclone-conf.yaml]: This configmap shouldn't be created at all if `rcloneConfig` is unset

**Comment by DanielZhangQD** [comment]
@mightyguava Thanks for fixing the errors.

**Pull request merged**